codes.json

```diff
@@ -364,6 +364,102 @@
       "en": "No groups names were provided."
     }
   },
+  "IncorrectNumber": {
+    "status": 402,
+    "code": 4020042,
+    "error": "incorrect_number",
+    "description": {
+      "en": "The card number is incorrect."
+    }
+  },
+  "InvalidNumber": {
+    "status": 402,
+    "code": 4020043,
+    "error": "invalid_number",
+    "description": {
+      "en": "The card number is not a valid credit card number."
+    }
+  },
+  "InvalidExpiryMonth": {
+    "status": 402,
+    "code": 4020044,
+    "error": "invalid_expiry_month",
+    "description": {
+      "en": "The card's expiration month is invalid."
+    }
+  },
+  "InvalidExpiryYear": {
+    "status": 402,
+    "code": 4020045,
+    "error": "invalid_expiry_year",
+    "description": {
+      "en": "The card's expiration year is invalid."
+    }
+  },
+  "InvalidCvc": {
+    "status": 402,
+    "code": 4020046,
+    "error": "invalid_cvc",
+    "description": {
+      "en": "The card's security code is invalid."
+    }
+  },
+  "ExpiredCard": {
+    "status": 402,
+    "code": 4020047,
+    "error": "expired_card",
+    "description": {
+      "en": "The card has expired."
+    }
+  },
+  "IncorrectCvc": {
+    "status": 402,
+    "code": 4020048,
+    "error": "incorrect_cvc",
+    "description": {
+      "en": "The card's security code is incorrect."
+    }
+  },
+  "IncorrectZip": {
+    "status": 402,
+    "code": 4020049,
+    "error": "incorrect_zip",
+    "description": {
+      "en": "The card's zip code failed validation."
+    }
+  },
+  "CardDeclined": {
+    "status": 402,
+    "code": 4020050,
+    "error": "card_declined",
+    "description": {
+      "en": "The card was declined."
+    }
+  },
+  "Missing": {
+    "status": 402,
+    "code": 4020051,
+    "error": "missing",
+    "description": {
+      "en": "There is no card on a customer that is being charged."
+    }
+  },
+  "ProcessingError": {
```

Comment:
This is the same codes of Stripe with the same names. Just I am using the same notation of them.